src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import { wait } from '../../utils/fetchClient';
+import { Error } from '../../types/Error';
+
+type Props = {
+  errors: Error[];
+  setErrors: (cb: (prev: Error[]) => Error[]) => void;
+};
+
+export const Notification: React.FC<Props> = ({ errors, setErrors }) => {
+  const [isHidden, setIsHidden] = useState(false);
+  const { title, isDanger } = errors[0];
+
+  const closeError = () => {
+    setIsHidden(true);
+    wait(900).then(() => {
+      setErrors(prev => prev.slice(1));
+      setIsHidden(false);
+    });
+  };
+
+  useEffect(() => {
```

Comment:
You are using 'setInterval' to close the error after 3 seconds, but you are not clearing the interval when the component is unmounted. Use 'clearInterval' inside a useEffect cleanup function to avoid potential memory leaks.